src/pages/TimeTablePage/TimeTable/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './style.scss';
+
+const TimeTable: React.FC = () => {
+
+    const arr: number[] = [];
+    for (let i = 0; i < 24; i++) {
+        arr.push( i );
+    }
+
+    const rows = arr.map((i: number) => (
+        <div key={i}>Child</div>
+    ));
+
+    return (
+        <div className="time-table">{rows}</div>
+    );
+};
+
+export default TimeTable;
```

Comment:
Purpose for this index.tsx ???

Reply:
its the code which is generating 24 rows in table
commited this, as it may be required later